functions

```diff
@@ -549,7 +551,9 @@ function create_kayobe_environment {
     env=$1
     mkdir -p "$env"/{src/kayobe,src/kayobe-config,venvs/kayobe}
     git clone -b $KAYOBE_BRANCH --single-branch $KAYOBE_URI "$env/src/kayobe"
-    cp -rf "$KAYOBE_CONFIG_SOURCE_PATH" "$env/src/kayobe-config"
+    sudo_if_available cp -rf "$KAYOBE_CONFIG_SOURCE_PATH" "$env/src/kayobe-config"
+    sudo_if_available chown -Rf stack:stack "$env/src/kayobe-config"
```

Comment:
Why stack user?

Reply:
Just that is the user we configure in the docker image:

https://github.com/stackhpc/kayobe-automation/blob/main/docker/kayobe/Dockerfile#L34

You are right though... maybe I should use $USER so that this also continues to work when not using the docker image?

Reply:
```suggestion
    sudo_if_available chown -Rf $USER:$USER "$env/src/kayobe-config"
```